plugin/sampling/strategystore/static/strategy_store.go

```diff
@@ -29,31 +32,103 @@ import (
 )
 
 type strategyStore struct {
+	// to allow concurrent update of sampling strategies
+	lock sync.RWMutex
+
 	logger *zap.Logger
 
 	defaultStrategy   *sampling.SamplingStrategyResponse
 	serviceStrategies map[string]*sampling.SamplingStrategyResponse
 }
 
+func (h *strategyStore) loadAndParseStrategies(strategiesFile string) error {
+	s, err := loadStrategies(strategiesFile)
+	if err != nil {
+		h.logger.Warn("Using the last saved configuration for sampling strategies.", zap.Error(err))
+		return err
+	}
+
+	h.logger.Info("Updating sampling strategies file!", zap.Any("Strategies", s))
```

Comment:
What is the intention of this log stmt? Are you trying to log the actual loaded strategies?

Reply:
Must have added this for debugging, removing from PR.